connector/pom.xml

```diff
@@ -456,6 +456,14 @@
                     <target>${java.target.version}</target>
                 </configuration>
             </plugin>
+            <!--
+            Perform the maven shade to have all the specific dependencies used in the connector
+            Use the most used libraries in the shade. This will ensure that the connector will not have any conflicts
```

Comment:
Accept if true :)
```suggestion
            Use the most (globally) used libraries in the shade. This will ensure that the connector will not have any conflicts
```